test/test_tree_2d_mhdmultiion.jl

```diff
@@ -0,0 +1,104 @@
+module TestExamples2DIdealGlmMhdMultiIon
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2d_dgsem")
```

Comment:
Why not
```suggestion
EXAMPLES_DIR = joinpath(examples_dir(), "tree_2d_dgsem")
```

Reply:
Done in 8af20590c534fde774ebe5721e1ccac0248d6013.